### PR TITLE
feat: add owner information

### PR DESCRIPTION
**Pull request by rsommerard**

### CODE REVIEW

File: bee/src/main/java/com/apisense/bee/ui/adapter/AvailableExperimentsRecyclerAdapter.java [hunk 1]
```diff
@@ -55,6 +55,7 @@ public void onBindViewHolder(ViewHolder holder, int position) {
         Crop crop = mAvailableCrops.get(position);
 
         holder.mCropTitle.setText(crop.getName());
+        holder.mCropOwner.setText("By " + crop.getOwner());
```
Comment (aveuiller) [inline bee/src/main/java/com/apisense/bee/ui/adapter/AvailableExperimentsRecyclerAdapter.java]: Use the internationalized String `R.string.exp_details_organization`

File: bee/src/main/java/com/apisense/bee/ui/adapter/SubscribedExperimentsRecyclerAdapter.java [hunk 2]
```diff
@@ -61,6 +61,7 @@ public void onBindViewHolder(SubscribedExperimentsRecyclerAdapter.ViewHolder hol
         Crop crop = mInstalledCrops.get(position);
 
         holder.mCropTitle.setText(crop.getName());
+        holder.mCropOwner.setText("By " + crop.getOwner());
```
Comment (aveuiller) [inline bee/src/main/java/com/apisense/bee/ui/adapter/SubscribedExperimentsRecyclerAdapter.java]: Use the internationalized String R.string.exp_details_organization

File: bee/src/main/res/layout/list_item_home_experiment.xml [hunk 3]
```diff
@@ -26,14 +26,22 @@
...
+        android:textSize="@dimen/secondary_text_size"
+        android:layout_below="@+id/crop_title"/>
+
+    <TextView
         android:id="@+id/crop_description"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/default_text_margin"
         android:textSize="@dimen/secondary_text_size"
         android:maxLines="2"
```
Comment (aveuiller) [inline bee/src/main/res/layout/list_item_home_experiment.xml]: Useless in this TextView?

File: bee/src/main/res/layout/list_item_home_experiment.xml [hunk 4]
```diff
@@ -26,14 +26,22 @@
         android:textSize="@dimen/primary_text_size"/>
 
     <TextView
+        android:id="@+id/crop_owner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/default_text_margin"
+        android:textSize="@dimen/secondary_text_size"
```
Comment (aveuiller) [inline bee/src/main/res/layout/list_item_home_experiment.xml]: Missing `android:textColor`?

**Comment by rsommerard** [comment]
Changes made :)

**Pull request merged**